network/hybrid/network.go

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+package hybridnetwork
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/iam"
+	"github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/to"
+)
+
+const (
+	errorPrefix = "Cannot create %v, reason: %v"
+)
+
+func getVnetClient(activeDirectoryEndpoint, tokenAudience string) network.VirtualNetworksClient {
+	token, err := iam.GetResourceManagementTokenHybrid(activeDirectoryEndpoint, tokenAudience, helpers.TenantID(), helpers.ClientID(), helpers.ClientSecret())
+	if err != nil {
+		log.Fatal(fmt.Sprintf(errorPrefix, "virtual network", fmt.Sprintf("Cannot generate token. Error details: %v.", err)))
+	}
+	vnetClient := network.NewVirtualNetworksClientWithBaseURI(helpers.ArmEndpoint(), helpers.SubscriptionID())
```

Comment:
>NewVirtualNetworksClientWithBaseURI [](start = 23, length = 35)

When creating clients, could you also add extra info to the user agent specifying this operations come from a sample? Take a look to what is already done for the non hybrid samples.

Reply:
Done